src/app/views/auth.go

```diff
@@ -445,19 +445,17 @@ func authGroup(router *gin.Engine) {
 				c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 				return
 			}
+			if err := newUser.ExpirePassword(ctx.(context.Context)); err != nil {
```

Comment:
no please add expire password for all creating users with null password on query